### ISSUE TITLE
Gallery creation

### ISSUE THREAD

**Issue by JessiJess** [gs-gs/swanningaround#47]
Hi, I would like to add a gallery to the website. A similar to the one that we have on this website: https://www.clipperroundtheworld.com/news/gallery. 

As you will see on the link above, the gallery is classified in folder and once the mouse is on the folder (without clicking on it) it reveals the theme and date. Then, you click on it and you have all the pictures related to the folder. See screenshot:

<img width="1059" alt="Screenshot - Gallery" src="https://user-images.githubusercontent.com/59960460/79938599-6a395f00-84a0-11ea-841b-e1ba790f5562.png">

Can you create the same interactive gallery for our website?

**Comment by JessiJess** [comment]
Hi @fritzelemino can you review the above request and come up with some design ideas similar to the one suggested above?

**Comment by fritzelemino** [comment]
Hi @JessiJess , 

Please check these:
https://www.dropbox.com/sh/dklfk6woex4rk5i/AABD1CVdS3ka1tLEurlF-PxIa?dl=0

**Comment by JessiJess** [comment]
Hi @fritzelemino , on the above screenshot, as you can see each folder have one picture with a file name so I would like something similar to classify our different adventure and location i.e. instead of 'Race 7: (...) 5 Mars 2020, it will be Passage Sydney to Bay of Island [Date] and when you click another sub gallery opens with all the related pictures. Is it what you proposed as I cannot really see?

**Comment by fritzelemino** [comment]
Hi @JessiJess , 

Kindly check the updated version here:
https://www.dropbox.com/sh/dklfk6woex4rk5i/AABD1CVdS3ka1tLEurlF-PxIa?dl=0

**Comment by JessiJess** [comment]
@fritzelemino  happy with this, thanks.

**Comment by JessiJess** [comment]
@fritzelemino , just checking, should I contact Ilya to implement your last version of the proposed gallery or is it something that you do?

**Comment by fritzelemino** [comment]
Hi @JessiJess , 

I only do design Ilya do all the implementation.

Thank you

**Comment by JessiJess** [comment]
Hi @webilhan, can you please implement the attached gallery to the website: https://www.dropbox.com/sh/dklfk6woex4rk5i/AABD1CVdS3ka1tLEurlF-PxIa?dl=0

**Comment by webilhan** [comment]
@JessiJess Do we really need filter "Show all" and "Search"? See what I mean on this screen - http://prntscr.com/saxa61

**Comment by JessiJess** [comment]
Hiv @webilhan , no indeed not necessary.

**Comment by JessiJess** [comment]
Hi @webilhan , do you have any update on the gallery creation?

**Comment by JessiJess** [comment]
Hi @webilhan , any updates?

**Comment by webilhan** [comment]
@JessiJess  Working on it.
@fritzelemino Could you share with me PSD files?

**Comment by JessiJess** [comment]
@fritzelemino can you please share the PSD files with Ilya?

**Comment by JessiJess** [comment]
@fritzelemino , please share the PSD file to @webilhan . Thanks

**Comment by JessiJess** [comment]
Hi @webilhan  and @fritzelemino , happy new year! Hope you had both nice festive season.
Can I check with you when this is at? I would like to add some pictures.
Thanks

**Comment by fritzelemino** [comment]
Hi @webilhan @JessiJess 

Please check PSD here: https://www.dropbox.com/sh/n98zf2u4fy13836/AAAl_JfpjEQENhmWDpr-Hm2Wa?dl=0

**Comment by fritzelemino** [comment]
@webilhan @JessiJess , I also noticed that the map has a **"For Development purposes only"** watermarks. I think we need to get a license key from Google map to remove this.

**Comment by webilhan** [comment]
@fritzelemino Yes, thanks for info. We already have [an issue](https://github.com/gs-gs/swanningaround/issues/63) for that.

**Comment by webilhan** [comment]
@JessiJess Finished with the gallery, you can check how it looks there - https://smooth-falcon.cloudvent.net/gallery/
Maybe we should add some back button from inner gallery page?

**Comment by JessiJess** [comment]
@webilhan , does look good and yes I agree we should be able to go back to the gallery page when we are inside a specific sub gallery.
